daemon/graphdriver/devmapper/README.md

```diff
@@ -92,5 +92,5 @@ This uses the `dm` prefix and would be used something like `docker daemon --stor
 
 These options are currently documented both in [the man
 page](../../../man/docker.1.md) and in [the online
-documentation](https://docs.docker.com/reference/commandline/daemon/#storage-driver-options).
+documentation](https://docs.docker.com/engine/userguide/storagedriver/device-mapper-driver/).
```

Comment:
Actually, this should be https://docs.docker.com/engine/reference/commandline/dockerd/#/storage-driver-options

@YuPengZTE are you interested in doing a follow up PR to change that?